.github/workflows/format.yml

```diff
@@ -15,18 +15,18 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        project: [engine, client, runtime, products/commandline]
+        project: [engine, client, products/commandline]
```

Comment:
we should be testing communication here too, no?

Reply:
yes, we should do so.